src/components/common/FileUploadPdf.vue

```diff
@@ -205,7 +207,7 @@ export default class FileUploadPdf extends Vue {
     try {
       const arrayBuffer = await file.arrayBuffer()
       const data = new Uint8Array(arrayBuffer) // put it in a Uint8Array
-      const document = await this.pdfjsLib.getDocument({ data: data })
+      const document = await this.pdfjsLib.getDocument({ data: data }).promise
```

Comment:
Sorry Karim, I don't mean to steal your thunder. I did this to fix my local testing. I can revert if you want.

Reply:
Hahahaha, no worries Sev! I've tried uploading a file (in the temporary URL) and it works fine! So, I'm thinking in my PR, I'm just going to rebase yours and add the unit tests that Travis requested.

Reply:
Awesome! Thanks for being gracious :)